usaspending_api/search/tests/test_spending_by_award_type.py

```diff
@@ -237,3 +237,54 @@ def test_spending_by_award_both_zip_filter(client, mock_matviews_qs):
         }))
     assert len(resp.data['results']) == 1
     assert resp.data['results'][0] == {'internal_id': 1, 'Place of Performance Zip5': '00001'}
+
+
+@pytest.mark.django_db
+def test_spending_by_award_foreign_filter(client, mock_matviews_qs):
+    """ Verify that foreign country filter is returning the correct results """
+    mock_model_1 = MockModel(award_id=1, piid=None, fain='abc', uri=None,
+                             type='B', pulled_from="AWARD", recipient_location_country_name="United States",
+                             recipient_location_country_code="USA")
+    mock_model_2 = MockModel(award_id=2, piid=None, fain='abd', uri=None,
+                             type='B', pulled_from="AWARD", recipient_location_country_name="United States",
```

Comment:
Is this ("United States") definitely lowercase? In the place of performance filter, its all uppercased...